frontend/packages/dev-console/integration-tests/features/monitoring/link-Kiali-ODC-3697.feature

```diff
@@ -0,0 +1,138 @@
+Feature: Link Kiali Dashboard
+    User should be able to link Kiali to namespace and visit the Kiali Dashboard from Topology page
+
+
+Background: 
+    Given user is at Administrator perspective
+
+
+@regression
+Scenario: Install ElasticSearch Operator
+    Given user is at OperatorHub page
+    When user searches for ElasticSearch Operator
+    And user clicks on the ElasticSearch Operator card
+    And user clicks on the Install button
+    And user selects 4.5 channel
+    And user selects default installation mode
+    And user selects automatic approval strategy
+    And user clicks on Install button
+    Then user will see a modal saying ElasticSearch Operator is installed
+
+
+@regression
+Scenario: Install Jaeger Operator
+    Given user is at OperatorHub page
+    When user searches for Jaeger Operator
+    And user clicks on the Jaeger Operator card
+    And user clicks on the Install button
+    And user selects stable channel
+    And user selects default installation mode
+    And user selects automatic approval strategy
+    And user clicks on Install button
+    Then user will see a modal saying Jaeger Operator is installed
+
+
+@regression
+Scenario: Install Kiali Operator
+    Given user is at OperatorHub page
+    When user searches for Kiali Operator
+    And user clicks on the Kiali Operator provided by Red Hat card
+    And user clicks on the Install button
+    And user selects stable channel
+    And user selects default installation mode
+    And user selects automatic approval strategy
+    And user clicks on Install button
+    Then user will see a modal saying Kiali Operator is installed
+
+
+@regression
+Scenario: Install Red Hat OpenShift Service Mesh Operator
+    Given user is at OperatorHub page
+    When user searches for Red Hat OpenShift Service Mesh Operator
+    And user clicks on the Red Hat OpenShift Service Mesh Operator card
+    And user clicks on the Install button
+    And user selects stable channel
+    And user selects default installation mode
+    And user selects automatic approval strategy
+    And user clicks on Install button
+    Then user will see a modal saying Jaeger Operator is installed
+
+
+@regression
+Scenario: Create New Project to link to Kiali Dashboard
```

Comment:
Scenario description and steps are not matching
In description, mentioned new link to kiali dashboard, but there is no steps related to kiali